compiler/circle-interpreter-test/src/circle-interpreter.test.cpp

```diff
@@ -0,0 +1,265 @@
+/*
```

Comment:
This code is mostly sourced from `circle-operator-test/src/circle-operator.test.cpp`.